features/step_definitions/generic_steps.rb

```diff
@@ -132,6 +132,10 @@ def assure_xpath_not_present(tag, attribute, value, selector)
   find_field("#{labeltext}").value.should == content
 end
 
+Then "I should see in the {string} input" do |labeltext, content|
+  find_field(labeltext).value.should == content
```

Comment:
How does this match two things out of one string param?

Reply:
I first left it undefined and cucumber suggested a stub for this.

It looks like this is what is happening:
https://cucumber.io/docs/gherkin/reference/#step-arguments
> In your step definition, there’s no need to find this text and match it in your pattern. It will automatically be passed as the last argument in the step definition.